### PR TITLE
Backend settings JSON schema

### PR DESCRIPTION
**Pull request by jolopezl**
JSON schema for user and system configuration support (see https://github.com/CARTAvis/carta-backend/issues/716).

### CODE REVIEW

**Comment by veggiesaurus** [comment]
@jolopezl can you update this schema with the latest changes? then we can close https://github.com/CARTAvis/carta-backend/issues/716

**Comment by jolopezl** [comment]
> @jolopezl can you update this schema with the latest changes? then we can close [CARTAvis/carta-backend#716](https://github.com/CARTAvis/carta-backend/issues/716)

Done.

File: schemas/preference_backend_schema_1.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            "type": "integer",
+            "default": -1
+        },
+        "read_only_mode": {
+            "type": "boolean",
+            "default": false
+        }
+    },
+    "additionalProperties": false,
+    "oneOf": []
```
Comment (veggiesaurus) [inline schemas/preference_backend_schema_1.json:85]: do we need this?

Comment (jolopezl) [reply schemas/preference_backend_schema_1.json:85]: Not needed at all, we can drop those lines. Will do that now.

**Pull request merged**